plugins/oauth/web_client/templates/oauth_login.jade

```diff
@@ -1,6 +1,6 @@
 .g-oauth-container
   .g-oauth-section-header
-    .g-oauth-section-header-text Or log in with
+    .g-oauth-section-header-text Or #{modeText} with
```

Comment:
This variable is never passed into the template.

Also, it's fine for it to say "log in" even in the register case, most sites seem to phrase it that way.